### PR TITLE
Test for STAC implementation

### PR DESCRIPTION
**Pull request by joaherrerama**

### CODE REVIEW

File: tests/test_async_processing_import_export.py [hunk 1]
```diff
@@ -323,6 +404,46 @@ def test_sentinel_import_error(self):
         self.waitAsyncStatusAssertHTTP(rv, headers=self.admin_auth_header,
                                        http_status=400, status="error")
 
+    # Test for STAC
+    @unittest.skipIf('GOOGLE_APPLICATION_CREDENTIALS' not in os.environ and 'GOOGLE_CLOUD_PROJECT' not in os.environ,
+                     "Test is skipped because 'GOOGLE_APPLICATION_CREDENTIALS' and 'GOOGLE_CLOUD_PROJECT' not set")
```
Comment (mmacata) [inline tests/test_async_processing_import_export.py:409]: `GOOGLE_APPLICATION_CREDENTIALS` and `GOOGLE_CLOUD_PROJECT` is not needed here. Instead, could you test if the stac plugin is installed and skip the tests if not? (Also valid for below tests).

File: tests/test_async_processing_import_export.py [hunk 2]
```diff
@@ -323,6 +404,46 @@ def test_sentinel_import_error(self):
...
+    def test_stac_import(self):
+
```
Comment (mmacata) [inline tests/test_async_processing_import_export.py:411]: Could you please add a sentence, saying what this test is doing? We agreed upon starting this for all new tests, so no great example until now, but then we have a good example ;)

**Review by mmacata — COMMENTED** [review]
Thank you for this update. I only have small comments.

File: tests/test_async_processing_import_export.py [hunk 3]
```diff
@@ -32,9 +32,15 @@
 except ModuleNotFoundError:
     from test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
 
+try:
+    import actinia_stac_plugin
+    no_stac_plugin = False
+except Exception:
+    no_stac_plugin = True
```
Comment (joaherrerama) [inline tests/test_async_processing_import_export.py:39]: This was the best option that cam up to my mind on how to verify the installation. Any suggestion to make it cleaner?

Comment (mmacata) [reply tests/test_async_processing_import_export.py:39]: Fine for me. What do you think @anikaweinmann ?

Comment (anikaweinmann) [reply tests/test_async_processing_import_export.py:39]: For me it is also fine

**Review by anikaweinmann — APPROVED** [review]

**Review by mmacata — APPROVED** [review]

**Pull request merged**